pkg/alertmanager/multitenant_test.go

```diff
@@ -43,38 +47,6 @@ receivers:
   - name: dummy`
 )
 
-// basic easily configurable mock
-type mockAlertStore struct {
```

Comment:
Removed in favour of using a real implementation (the one backed by filesystem) or the `bucket.ClientMock` when we need to mock the backend (eg. simulate a failure).

Reply:
👍